docker/docker-compose.yml

```diff
@@ -19,16 +19,20 @@ services:
       - postgres
       - rabbitmq
     networks:
-      - stb-backend
-      - stb-frontend
+      stb-backend:
+        aliases:
+          # Use alias here because Chrome (and not only Chrome) supports HSTS in strange way.
```

Comment:
I guess that it isn't required.
We pass the host address instead of the host name to the selenium, so it probably won't be HSTS preloaded
Perhaps, it will be better to keep this in case we return to host name. Decide

Reply:
@artemiy312 , y, it became non actual because of your's django grading task.
I'll remove it.

If we'll return aliases, we'll search this problem at our repo for the first. So, code don't need this notes, i think